Automattic/wp-calypso
# Fix home link accessible name in Jetpack masterbar

### Changes proposed in this Pull Request

This PR updates the name read by screen readers for the home link of the Jetpack masterbar in cloud, from _Jetpack_ to _Jetpack Cloud Dashboard_. The latter is more accurate and is what is shown when hovering the link with the mouse pointer.

Fixes 1164141197617539-as-1200033729320427

### Implementation notes

The `title` of the Jetpack logo overwrites the link accessible name. Adding the attribute `aria-hidden` to the logo `svg` tag makes it invisible to screen readers.

### Testing instructions

- Download the PR and run cloud
- Visit any page
- Check in the inspector that the svg of the Jetpack logo in the masterbar has the attribute `aria-hidden` set to `true`
<img width="676" alt="Screen Shot 2021-03-08 at 4 15 47 PM" src="https://user-images.githubusercontent.com/1620183/110382882-86e37480-8029-11eb-9791-64de47f0755b.png">


_Note: the following assumes you're using VoiceOver on OSX. Instructions may differ with other screen readers._

If you want to go a step further:
- Activate VoiceOver (`⌘+F5`)
- Open the Web Rotor (`Control+Option+U`). The rotor is a handy way to view the page main features (such as headings, links, etc.) and provides shortcuts for navigating.
- Check that first item in the _Links_ section is _Jetpack Cloud Dashboard_

### Screenshots

_Before_

https://user-images.githubusercontent.com/1620183/110382416-e0976f00-8028-11eb-89a0-fca7eb50c7f2.mov



_After_

https://user-images.githubusercontent.com/1620183/110382428-e68d5000-8028-11eb-98d5-f1638ce29a58.mov
